grpc.gemspec

```diff
@@ -16,17 +16,20 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.0.0'
   s.requirements << 'libgrpc ~> 0.11.0 needs to be installed'
 
-  s.files = %w( Rakefile )
-  s.files += Dir.glob('bin/**/*')
-  s.files += Dir.glob('ext/**/*')
-  s.files += Dir.glob('lib/**/*')
-  s.files += Dir.glob('pb/**/*')
-  s.test_files = Dir.glob('spec/**/*')
+  s.files = %w( Rakefile Makefile )
+  s.files += Dir.glob('src/ruby/bin/**/*')
+  s.files += Dir.glob('src/ruby/ext/**/*')
+  s.files += Dir.glob('src/ruby/lib/**/*')
+  s.files += Dir.glob('src/ruby/pb/**/*')
+  s.files += Dir.glob('include/grpc/**/*')
+  s.files += Dir.glob('src/core/**/*')
```

Comment:
Can we template this?

There have been cases (and likely will be cases again) where we have code in src/core that we don't necessarily want to bundle up for distribution, but nor do we want to delete it.